contracts/mixnet/src/helpers.rs

```diff
@@ -11,6 +15,9 @@ const HOURS_IN_YEAR: u128 = 8760;
 
 const DECIMAL_FRACTIONAL: Uint128 = Uint128(1_000_000_000_000_000_000u128);
 
+// cosmwasm bucket internal value
+const NAMESPACE_LENGTH: usize = 2;
```

Comment:
Is there no way to avoid using this? This looks like a ticking timebomb that will bite us in the future

Reply:
Unfortunately I don't think there is much we can do about it. I think this namespace prefix appears because the node identity is part of the namespace of the bucket. One possible solution would be to move it into the key space, where the owner address also resides, by concatenation. I _think_ that might make this hardcoded value go away, but I'm not sure it's possible to iterate over partial keys the way I iterated here over partial namespaces. In any case, moving the node identity from namespace to key space would require yet another migration, so I definitely wouldn't do that now, that the delegations are in a somewhat weird state.
On the bright side, if anything changes in the internals of cosmwasm bucket regarding this, our unit tests should catch it.